pkg/controller/backup_controller.go

```diff
@@ -476,7 +477,7 @@ func (b *backupReconciler) prepareBackupRequest(backup *velerov1api.Backup, logg
 		request.Status.ValidationErrors = append(request.Status.ValidationErrors, "encountered labelSelector as well as orLabelSelectors in backup spec, only one can be specified")
 	}
 
-	if request.Spec.ResourcePolicy != nil && request.Spec.ResourcePolicy.Kind == resourcepolicies.ConfigmapRefType {
+	if request.Spec.ResourcePolicy != nil && strings.EqualFold(request.Spec.ResourcePolicy.Kind, resourcepolicies.ConfigmapRefType) {
```

Comment:
Is it possible to add a simple test just to check this new behavior?

Reply:
Updated the test case for resourcemodifier, that should be enough coverage.
For resourcepolicies, it is not trivial to add for now, since there are no controller side tests from what I can see.